layers/meta-resin-allwinner/conf/machine/orangepi-plus2e.conf

```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
+#@NAME: Orange Pi Plus 2e
+#@DESCRIPTION: Machine configuration for the Orange Pi Plus2e, based on allwinner H3 CPU http://www.orangepi.org/
+
+require conf/machine/include/sun8i.inc
```

Comment:
You should define instead a machine override like in: https://github.com/balena-os/balena-allwinner/blob/master/layers/meta-resin-allwinner/conf/machine/bananapi-m1-plus.conf#L5